projects/angular/src/layout/tabs/STYLE.md

```diff
@@ -0,0 +1,21 @@
+# Tabs
+
+## CSS Custom Properties
+
+| CSS Custom Property               | Description                       |
+| --------------------------------- | --------------------------------- |
+| --clr-nav-link-active-font-weight | Tabs nav link active font weight. |
+| --clr-nav-link-font-weight        | Tabs nav link font weight.        |
+| --clr-nav-link-active-color       | Tabs nav link active color.       |
+| --clr-nav-hover-bg-color          | Tabs nav hover background color.  |
+| --clr-nav-active-bg-color         | Tabs nav active background color. |
+| --clr-nav-box-shadow-color        | Tabs nav box shadow color.        |
+| --clr-nav-active-box-shadow-color | Tabs nav active box shadow color. |
```

Comment:
`clr-nav-link-color` missed from properties.

Reply:
corrected in commit 8830c51